tests/batcomputer_cli/commands/identify.py

```diff
@@ -24,7 +24,9 @@ def identify(
     """
     responses = [
         (
-            get_oracle_data(alias) if oracle else get_alias_data(alias),
+            get_oracle_data(alias)
```

Comment:
Here we also need to add lower, right?

Like this, still breaks when we run `tests/batcomputer.py -o id Batman`, for example

Reply:
Yes! This PR fix only one use case. 

If you like, I can do it in the PR as well.

Reply:
It is very related, no need for an other PR